### ISSUE TITLE
brave pkg help

### ISSUE THREAD

**Issue by tnxz** [mtoohey31/nixexprs#1]
hey I have came across your brave pkg trying to find a brave overlay
but I seem to be getting the following error after install
any idea why these error pops or is it intended
<img width="372" alt="Screenshot 2023-09-10 at 11 00 12 AM" src="https://github.com/mtoohey31/nixexprs/assets/91027295/012fdc98-aaf2-4325-94a8-a735bec11d81">

Darwin configuration.nix
```
  nixpkgs = {
    overlays = [
      (inputs: pkgs:
        let inherit (pkgs) callPackage;
        in { brave = callPackage ./brave.nix { inherit (pkgs) brave; }; })
    ];
  };

```

brave.nix
```
{ brave, fetchurl, stdenv, undmg }:

if stdenv.hostPlatform.isDarwin then
  stdenv.mkDerivation
  rec {
    pname = "brave";
    inherit (brave) version;
    sourceRoot = "Brave Browser.app";
    src = fetchurl {
      url = "https://github.com/brave/brave-browser/releases/download/v${version}/Brave-Browser-universal.dmg";
      sha256 = "sha256-amddEX2AXBjc4etXrdFtLrMMrYPVEwUhqq1m/b4kv0M=";
    };
    buildInputs = [ undmg ];
    installPhase = ''
      mkdir -p $out/Applications/Brave\ Browser.app
      cp -R . $out/Applications/Brave\ Browser.app
    '';
  } else brave

```

**Comment by mtoohey31** [comment]
Hmm, I haven't seen that error myself. I wonder if it's maybe something for specific MacOS versions, or something like that. What version of MacOS was this with?

**Comment by tnxz** [comment]
thanks for responding quickly
```
$ sw_vers 
ProductName:		macOS
ProductVersion:		13.5.2
BuildVersion:		22G91
```
mac m1 air

**Comment by mtoohey31** [comment]
This check might be something that's new in MacOS 13 then. I'm running MacOS 12 on the only Mac that I use the derivation on, and it's not new enough to support 13, so I don't have any way of testing this. Some searching seemed to indicate that people have seen similar warnings before 13 though, but I didn't see any conclusion as to what caused it... So I'm really not sure why you're seeing this.

**Comment by tnxz** [comment]
okay thanks any way let me do some digging if i find anything i will drop it here.

**Comment by tnxz** [comment]
Figured it out [https://github.com/NixOS/nixpkgs/commit/3ea22dab7d906f400cc5983874dbadeb8127c662](https://github.com/NixOS/nixpkgs/commit/3ea22dab7d906f400cc5983874dbadeb8127c662)
```
{ brave, fetchurl, stdenv, undmg }:

if stdenv.hostPlatform.isDarwin then
  stdenv.mkDerivation
  rec {
    pname = "brave";
    inherit (brave) version;
    sourceRoot = "Brave Browser.app";
    src = fetchurl {
      url = "https://github.com/brave/brave-browser/releases/download/v${version}/Brave-Browser-universal.dmg";
      sha256 = "sha256-amddEX2AXBjc4etXrdFtLrMMrYPVEwUhqq1m/b4kv0M=";
    };
    buildInputs = [ undmg ];
    dontFixup = true;
    installPhase = ''
      mkdir -p $out/Applications/Brave\ Browser.app
      cp -R . $out/Applications/Brave\ Browser.app
    '';
  } else brave
```

**Comment by mtoohey31** [comment]
Great, thanks! I've added the same fix in e95a1c6b056862b1ca80a455fb42e156bdf44645.
